code/modules/mob/living/deity/deity.dm

```diff
@@ -16,7 +16,8 @@
 	bone_material = null
 	bone_amount = 0
 
-	var/eye_type = /mob/observer/eye/cult
+	var/eye_type = /mob/observer/eye/freelook/cult
+	var/mob/observer/eye/freelook/cult/eye
```

Comment:
why adding this var instead of using existing eyeobj? You're setting eyeobj anyway

Reply:
eyeobj is of type mob/observer/eye, rather than /mob/observer/eye/freelook, so it doesn't have the vars associated with visualnets. I figured this was cleaner than type-casting it every time I needed to access a visual net specific var, although thinking about it now it could just hold its visualnet datum instead.

Reply:
Typecasting is fine, IMO better than keeping second reference to existing var
Also it used to have visualnet var didn't it, just keep em.

Reply:
Addressing the second half, I meant that the deity would hold its visualnet datum. The vast majority of the references to the visual net itself through linked.eye.visualnet etc, so that will end up cleaner regardless.